sdk/js/src/sui/utils.ts

```diff
@@ -27,6 +31,27 @@ export const executeTransactionBlock = async (
   });
 };
 
+// TODO: can we pass in the latest core bridge package Id after an upgrade?
+// or do we have to use the first one?
+// this is the same type that the guardian will look for
+export const getEmitterAddressAndSequenceFromResponseSui = (
+  coreBridgePackageId: string,
```

Comment:
if we leave this in we should rename to `originalCoreBridgePackageId` for clarity